policyengine_us/variables/gov/states/id/tax/income/credits/grocery/id_grocery_credit_eligible.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for the Idaho grocery credit"
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(person, period, parameters):
+        spm_unit = person.spm_unit
+        # https://www.law.cornell.edu/uscode/text/42/1786#d_2_A_ii
+        snap_ineligible = add(spm_unit, period, ["snap"]) > 0
```

Comment:
this variable is named backwards. also the test is about receipt, not eligibility.

let's just remove the snap piece for now since the law says it's prorated for months without snap receipt